src/lib/chips/_chips-theme.scss

```diff
@@ -6,27 +7,36 @@
   $accent: map-get($theme, accent);
   $warn: map-get($theme, warn);
   $background: map-get($theme, background);
+  $foreground: map-get($theme, foreground);
+
+  $unselected-background: if($is-dark-theme, md-color($background, card), #e0e0e0);
+  $unselected-foreground: if($is-dark-theme, md-color($foreground, text), rgba(0, 0, 0, 0.87));
```

Comment:
Why are these not just
```scss
$unselected-background: md-color($background, card);
$unselected-foreground: md-color($foreground, text);
// etc.
```
?

The background and foreground colors are already based on whether it's a light/dark theme. Is there some reason the light theme version of these colors aren't appropriate?